internal/pkg/manifest/manifest.go

```diff
@@ -335,11 +335,30 @@ func (m *Manifest) GetParent(record model.Record) (model.Record, error) {
 }
 
 func (m *Manifest) ResolveParentPath(record model.Record) error {
+	return m.doResolveParentPath(record, nil)
+}
+
+// doResolveParentPath recursive + fail on cyclic relations.
+func (m *Manifest) doResolveParentPath(record, origin model.Record) error {
+	if origin != nil && record.Key().String() == origin.Key().String() {
+		return fmt.Errorf(`a cyclic relation was found when resolving path to %s`, origin.Desc())
+	}
+
+	if origin == nil {
+		origin = record
```

Comment:
Pri prvom volani funkcie s `origin  = nil` si poznacime, kde sme zacali.
Ak sa opat dostatneme k rovnakemu objektu, tak to signalizuje zacyklenie, vid kod vyssie.